torch/distributed/rpc/__init__.py

```diff
@@ -19,6 +19,7 @@ def is_available():
 
 if is_available():
     from .api import _init_rpc_backend, _require_initialized
+    from .api import _rpc_sync_torchscript, _rpc_async_torchscript
```

Comment:
Is this an unnecessary import? 
Users are not expected to use API starting with underscore, right?

Reply:
but training toolkit will use it?